packages/network/src/protocol/TrackerNode.ts

```diff
@@ -134,11 +136,11 @@ export class TrackerNode extends EventEmitter {
         return requestId
     }
 
-    async send<T>(receiverNodeId: string, message: T & TrackerLayer.TrackerMessage): Promise<void> {
-        await this.endpoint.send(receiverNodeId, message.serialize())
+    async send<T>(receiverPeerId: PeerId, message: T & TrackerLayer.TrackerMessage): Promise<void> {
```

Comment:
Just a thought, the receiver here should always be a tracker, see second parameter type. However, once we move towards a decentralized tracker algorithm the receiver would be a super node (or a node with a public ip)

Given that in most cases in this class the "PeerId" is "TrackerId" maybe it's the better option for now.

Reply:
Changed to `TrackerId`